### PR TITLE
Replace sprite svg icons with heroicons alternatives

### LINKED ISSUE #3703: Replace all svg icons in "sprites.html" with heroicons

### PR DESCRIPTION
**Pull request by theskumar**
- Upgrade heroicons to 2.6.0, adds support for heroicons_mirco
- Remove/upgrade sprite icons
- Update icons in the global messages widget
- Remove unused bgg-logo sprite
- Update calendar icon with heroicon
- Update anchor-link icon
- Remove speech-bubble icon and link—-open-feed, link--close-feed and link—to-top
- Remove arrow-up and arrow-down
- Remove arrow-split
- Remove "website" icon
- Remove “error” icon
- Update like & dislike button in the screening section
 - Remove 'bulb' svg from sprite
 - Remove 'hero-home-left-pixels' sprite
 - Remove 'hero-home-right-pixels' from sprite
 - Remove 'magnifying-glass' svg from sprite
-  Remove 'person-icon' and profile_button.html
 - Replace 'caret-down' and 'caret-up' icon
 - Remove 'side-arrow' from svg sprite
 - Remove 'add-user' icon, replace with 'user-plus'
 - Remove 'logo-desktop', 'logo-mobile' and 'logo-desktop--dark' icons
 - Remove 'logo-mobile-no-text' icon
 - Remove 'logo-mobile-dark' and 'logo-desktop--dark-alt'
 - Remove 'hero-standard-left-pixels' and 'hero-standard-right-pixels'
 - Remove 'eye' icon and replace with heroicon 'eye'
 - Remove 'exclamation-point' icon and replace by heroicon exlaimation-circle
 - Remove 'github' icon
 - Remove 'person-icon' and profile_button.html
 - Replace 'caret-down' and 'caret-up' icon
 - Remove 'side-arrow' from svg sprite

closes #3703 
depends on #3722

### CODE REVIEW

**Review by frjo — APPROVED** [review]
Will be nice to have one single source for all icons. They will all have the same look and there is no guessing where they come from.

If we in the future want to change icon set it will be easier as well.

**Comment by theskumar** [comment]
> Will be nice to have one single source for all icons. They will all have the same look and there is no guessing where they come from.

I didn't fully understand this. Did you intend to say not use heroicons? As for the guess work, I believe heroicons template tags avoids the guess work, and is very explicit where the icons came from. They are also quite isolated for each use, and can be easily updated with find and replace. 

For the custom icons, I was thinking to use web components and be included with a template tag. Thoughts?

**Comment by frjo** [comment]
I ment that the heroicons is a good single source for all icons.

**Comment by theskumar** [comment]
Rebased on top of #3722 to avoid merge conflicts.

**Comment by frjo** [comment]
I have merged in all "Remove public…" PRs before this one now so this can be rebased of main.

**Comment by theskumar** [comment]
Awesome, will do it now.

File: hypha/apply/home/templates/apply_home/includes/apply_listing.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
-{% load i18n bleach_tags markdown_tags wagtailcore_tags %}
```
Comment (frjo) [inline hypha/apply/home/templates/apply_home/includes/apply_listing.html:1]: I think ´ markdown_tags´  are needed. I added `{{ page.description|markdown|bleach }}`.

**Review by frjo — CHANGES_REQUESTED** [review]
One minor issue.

File: hypha/apply/home/templates/apply_home/includes/apply_listing.html (hunk 1 above)
Comment (theskumar) [reply hypha/apply/home/templates/apply_home/includes/apply_listing.html:1]: I agree.

File: hypha/apply/home/templates/apply_home/includes/apply_listing.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+{% load i18n bleach_tags wagtailcore_tags heroicons %}
```
Comment (theskumar) [inline hypha/apply/home/templates/apply_home/includes/apply_listing.html]:
```suggestion
{% load i18n bleach_tags wagtailcore_tags markdown_tags heroicons %}
```

File: hypha/public/utils/models.py
Comment (frjo) [inline hypha/public/utils/models.py:1]: Looks like we need this file for a bit longer.

**Review by frjo — CHANGES_REQUESTED** [review]

File: hypha/public/utils/models.py
Comment (theskumar) [reply hypha/public/utils/models.py:1]: Added back, got removed while rebasing

**Review by frjo — APPROVED** [review]

**Pull request merged**